### PR TITLE
Get results as CSV

### PR DESCRIPTION
**Pull request by AstroMatt**

### CODE REVIEW

File: atlassian/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.11.19
```
[the remark this answers is not in the document]
Comment (AstroMatt) [reply atlassian/VERSION]: It's a new feature and according to semantic versioning which is used here, new features should go with minor releases not bugfix (patch).

File: atlassian/jira.py [hunk 2]
```diff
@@ -51,6 +51,16 @@ def jql(self, jql, fields='*all', start=0, limit=None):
             params['jql'] = jql
         return self.get('rest/api/2/search', params=params)
 
+    def csv(self, jql, limit=99999):
```
[the remark this answers is not in the document]
Comment (AstroMatt) [reply atlassian/jira.py]: When `limit` is set to `None` I presume that there is no limit, while Jira in fact defaults limit to 1000.

Maybe 99999 is a bad idea, and we should keep Jira's default.
I will change that to 1000.

File: atlassian/VERSION (hunk 1 above)
[the remark this answers is not in the document]
Comment (gonchik) [reply atlassian/VERSION]: I hope releases will increase so faster, (1.2123.1 )

[the remark this answers is not in the document]
Comment (gonchik) [reply atlassian/VERSION]: " Major version zero is all about rapid development. If you’re changing the API every day you should either still be in version 0.y.z or on a separate development branch working on the next major version " 


Therefore let's use development branch

**Pull request merged**